### PR TITLE
#1805: TkJoinTest is not checking if join page is being rendered correctly

### PR DESCRIPTION
**Pull request by sbzDev**
Original issue is https://github.com/zerocracy/farm/issues/1805
I am newcomer, my resume was sent second time couple days ago. @g4s8 has posted the answer when I have asked about status of my resume first time:
> Check http://0crat.com/join page, your resume should be here
So I have decided to dig a src and fix a problem.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#2052](https://github.com/zerocracy/farm/issues/2052) is now in scope, role is `REV`

File: src/test/java/com/zerocracy/tk/TkJoinTest.java [hunk 1]
```diff
@@ -49,16 +51,16 @@
...
-                new RsPrint(
-                    new TkApp(farm).act(
-                        new RqWithUser.WithInit(
-                            farm, new RqFake("GET", "/join")
-                        )
-                    )
-                ).printBody()
-            ),
-            XhtmlMatchers.hasXPaths("//xhtml:body")
+                XhtmlMatchers.xhtml(
```
Comment (g4s8) [inline src/test/java/com/zerocracy/tk/TkJoinTest.java]: @sbzDev looks like unneeded formatting was applied here.

File: src/test/java/com/zerocracy/tk/TkJoinTest.java [hunk 2]
```diff
@@ -69,27 +71,49 @@ public void renderJoinPage() throws Exception {
      * @throws IOException If something goes wrong accessing page
      */
     @Test
-    @Ignore
```
Comment (g4s8) [inline src/test/java/com/zerocracy/tk/TkJoinTest.java]: @sbzDev you also need to remove todo puzzle with this change to complete the job: https://github.com/zerocracy/farm/blob/317bd2c140a0149abfe073597b098927d3d9ea8d/src/test/java/com/zerocracy/tk/TkJoinTest.java#L38-L41

File: src/main/resources/xsl/resume.xsl [hunk 3]
```diff
@@ -25,6 +25,46 @@ SOFTWARE.
...
+            </a>
+          </li>
+          <li>
+            <xsl:text>personality: </xsl:text>
+            <code>
+              <xsl:value-of select="personality"/>
+            </code>
+          </li>
+          <li>
+            <xsl:text>examiner: </xsl:text>
```
Comment (g4s8) [inline src/main/resources/xsl/resume.xsl]: @sbzDev `examiner` is optional element here, examiner is assigned automatically each hour, so when user submits the resume he/she doesn't have examiner assigned. So we need to check that element with `<xsl:if>` or `<xsl:choose>` before rendering.

**Comment by sbzDev** [comment]
@g4s8  I am waiting for your review of my second commit with fixes, right?

**Comment by g4s8** [comment]
@sbzDev thanks

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#2052` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#2052](https://github.com/zerocracy/farm/issues/2052) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**